src/python/RucioUtils.py

```diff
@@ -26,14 +27,21 @@ def getNativeRucioClient(config=None, logger=None):
     cl = logging.getLogger('charset_normalizer')
     cl.setLevel(logging.ERROR)
 
-    rucioCert = getattr(config.Services, "Rucio_cert", config.TaskWorker.cmscert)
-    rucioKey = getattr(config.Services, "Rucio_key", config.TaskWorker.cmskey)
+    # allow for both old and new configuration style
+
+    if getattr(config, 'Services', None):
+        rucioConfig = config.Services
+    else:
+        rucioConfig = config
```

Comment:
```suggestion
        rucioConfig = config.Rucio
```
? 
Ref: ![Screenshot from 2024-06-26 13-28-19](https://github.com/dmwm/CRABServer/assets/2346664/4901a6d4-5c72-45cc-908d-c03310cfd7f1)

Reply:
My idea is to pass `config.Rucio` to getNativeRucioClient, instead of the full `config` object. It seems a better encapsulation to me. RucioUtils does not need to know all the details for the full `config`.

Reply:
Ah..I see.
I missed the change in line 71 :facepalm: